fairscale/nn/wrap/auto_wrap.py

```diff
@@ -4,23 +4,77 @@
 # LICENSE file in the root directory of this source tree.
 
 import contextlib
-from typing import Any, Callable, Dict, Generator, List, Optional, Tuple
+from typing import Any, Callable, Dict, Generator, Optional, Set, Tuple, Type, cast
 
 import torch.nn as nn
 
-from fairscale.nn.data_parallel.fully_sharded_data_parallel import FullyShardedDataParallel
-from fairscale.nn.misc import checkpoint_wrapper
 
-# Modules that don't wrap.
-FSDP_MODULE_EXCLUDE_WRAP = {nn.ModuleList, nn.ModuleDict}
-# Modules that we don't recurse down to their children.
-FSDP_MODULE_BLOCKLIST = {nn.MultiheadAttention}
+def default_should_wrap_policy(
+    recurse: bool,
+    module: nn.Module,
```

Comment:
minor nit: what do you think about swapping the order of these two? In some sense, `module` is the main argument to this function and `recurse` is secondary

Reply:
done